src/server/schema/resolvers/queries/listLives.ts

```diff
@@ -0,0 +1,10 @@
+import { getDatabaseContext } from '../../../database';
+import { GraphQLQueryResolvers } from '../definitions';
+
+const query: GraphQLQueryResolvers['listLives'] = async ( root ) => {
+    const { collections } = await getDatabaseContext();
+
+    return collections.lives.find().map(item => {return {...item, fullName: `${item.firstName} ${item.lastName}`}}).toArray();
+};
+
+export default query;
```

Comment:
You should be careful, in many files you didn't provide a proper EOF (End-Of-File) which should be a last empty line. On GitHub when doing a diff, wrong EOF will be shown with a red marker.